variants/m5stack_atom/pins_arduino.h

```diff
@@ -3,6 +3,14 @@
 
 #include <stdint.h>
 
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       16
+
+#define analogInputToDigitalPin(p) (((p) < 20) ? (esp32_adc2gpio[(p)]) : -1)
+#define digitalPinToInterrupt(p)   (((p) < 40) ? (p) : -1)
+#define digitalPinHasPWM(p)        (p < 34)
+
```

Comment:
Hi @Tinyu-Zhao, those defines are in 3.X version defined globally per SoC in Arduino.h file.
Defining them here will trigger redefinition error/warning. Can you please remove those from the PR? Thanks
```suggestion
```

Reply:
I would like to ask, the constants defined in Arduino.h will not be added again in variants, right?

<img width="1148" alt="image" src="https://github.com/user-attachments/assets/04912a11-5376-48f9-b948-86559853682b">

Reply:
No they won't :)

Reply:
> No they won't :)

PTAL🙏